src/components/Settings/SettingsAddressbook.vue

```diff
@@ -54,6 +54,12 @@
 			</ActionLink>
 
 			<template v-if="!addressbook.readOnly">
+				<!-- sync profiles from socialmedia -->
+				<ActionLink
+					:href="avatarSettingsPage"
+					icon="icon-picture">
```

Comment:
Just created a sync icon there based on our already existing ones: https://gist.github.com/skjnldsv/035d5e723d03f64911ddec0d2af76c19
Feel free to add this icon to the img folder and into the icons.scss file :wink:

Reply:
Thinking about this, a sync-icon might imply bi-directional data flow. What do you think?

Reply:
I think it's better than the picture icon, but we can ask @jancborchardt here :)

Reply:
what about the globe? 
![grafik](https://user-images.githubusercontent.com/34400929/79617689-5be9ec80-8108-11ea-980b-dad826d1a6ab.png)

Reply:
No, the sync icon is more straightforward I'd say :)

Reply:
The issue here is not so much the icon, see my comment below at https://github.com/nextcloud/contacts/pull/1580#issuecomment-617131043 :)

Reply:
The sync options should be as long as there is a compatible social entries :)
Though we sill don't support setting a PREF as mentionned in https://github.com/nextcloud/contacts/issues/1594#issuecomment-617200825. So let's take the first one for now if there are multiple same social entries, we can improve later.

Reply:
I changed it for you, its the :arrows_counterclockwise: now :)